app/views/shared/_batch.html.erb

```diff
@@ -1,99 +1,78 @@
 
-  <table class="sortable table">
-    <thead>
-      <tr>
-        <% if edit -%>
-          <th width="3%">Drag</th>
-        <% elsif stage %>
-          <th class="nosort" width="7%"><input type='checkbox' class='select_all'></th>
+<table  id="batch-<%= edit ? 'edit' : 'show' %>"
+        class="table"
+        data-batch-id="<%= @batch&.id %>">
+  <thead>
+    <tr>
+      <% if stage %>
+        <th class="nosort" width="7%"><input type='checkbox' class='select_all'></th>
+      <% end -%>
+      <th>Position</th>
+      <th>Sample</th>
+      <th>Request</th>
+      <th>Source</th>
+      <th>Source barcode</th>
+      <th>Target Type</th>
+      <th>Target barcode</th>
+      <% if @batch.display_tags? %>
+        <th>Tag #</th>
+      <% end %>
+      <% @rits.each do |rit| %>
+        <th><%= rit.label %></th>
+      <% end %>
+      <th>State</th>
+      <th>Comments</th>
+    </tr>
+  </thead>
+  <tbody id="requests_list">
+    <% requests.each_with_index do |request, i| -%>
+      <tr id="request_<%= request.id %>" class="row<%= i %>" data-request-id="<%= request.id %>">
+
+        <% if stage %>
+            <td><input name='request[<%= request.id %>]' id="sample-<%= i+1  %>-checkbox" class='sample_check select_all_target' value="on" type='checkbox' checked></td>
         <% end -%>
-        <th>Position</th>
-        <th>Sample</th>
-        <th>Request</th>
-        <th>Source</th>
-        <th>Source barcode</th>
-        <th>Target Type</th>
-        <th>Target barcode</th>
-        <% if @batch.display_tags? %>
-          <th>Tag #</th>
+
+        <td><%= request.position %></td>
+        <td><%= request.asset&.display_name %></td>
+        <td><%= link_to request.id, request_url(request) %></td>
+
+        <% if request.asset  %>
+          <td><%= link_to request.asset.sti_type, receptacle_path(request.asset) %></td>
+          <td><%= request.asset.try(:human_barcode) %></td>
+        <% else %>
+          <td></td>
+          <td></td>
         <% end %>
-        <% @rits.each do |rit| %>
-          <th><%= rit.label %></th>
+
+        <% if request.target_asset %>
+          <td><%= link_to request.target_asset.sti_type, receptacle_path(request.target_asset) %></td>
+          <td><%= request.target_asset.try(:human_barcode) %></td>
+        <% else %>
+          <td></td>
+          <td></td>
         <% end %>
```

Comment:
Get rid of the nested if here and simplify. It does slightly change the behaviour (before we'd hide a target asset if we didn't have an asset, now we show it) but this feels more useful.